Sources/ProjectDescription/CopyFilesAction.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+public struct CopyFilesAction: Equatable, Codable {
+    public var name: String
+    public var destination: Destination
+    public var subpath: String
+    public var files: [FileElement]
```

Comment:
I'd document every public element in this file. That way, developers can get inline documentation when editing the manifest with `tuist edit`.